Specification.md

```diff
@@ -148,43 +184,57 @@ Nodes can either be in _Steady State_, or in _Suppression State_
 
 ## 5. Examples
 
+In the following examples, if the timestamp is not explicitly mentioned, it is assumed to be some aribitrary time.
+
 ### 5.1 State Sync - Example without packet loss
 
-Sync Group with 3 participants, node A, B, and C.
+Sync Group with 3 participants, node `A`, `B`, and `C`.
 
-- Data set state: [A=10, B=15, C=25], all nodes are in Sync
-- Node A publishes new publication.
-- A sends Sync Interests with [A=11, B=15, C=25]
-- B and C receive Sync Interest and update their local states accordingly.
+- Data set state: `[A=10, B=15, C=25]`, all nodes are in Sync
```

Comment:
Do you want to reflect bootstrap time here?

Reply:
Don't want to confuse. There's a note at the top.

```
if the timestamp is not explicitly mentioned, it is assumed to be some aribitrary time
```